typescript/test/unit/baseline.ts

```diff
@@ -46,15 +58,38 @@ const CLIENT_LIBRARY_BASELINE = path.join(
   'typescript',
   'test',
   'testdata',
+  'showcase',
+  'src',
   'echo_client_baseline.ts.txt'
 );
+
+const CLIENT_CONFIG_BASELINE = path.join(
+  cwd,
+  'typescript',
+  'test',
+  'testdata',
+  'showcase',
+  'src',
+  'echo_client_config.json.txt'
+);
+
+const CLIENT_PROTO_LIST_BASELINE = path.join(
```

Comment:
OK so this works, now let's make one more logical move and let's just put all the baseline files into the same folder structure and compare all of them.

```
testdata/
  showcase/
    package.json.txt
    src/
      index.ts.txt
        v1/
          echo_client.ts.txt
```

(maybe even rename `.txt` to `.baseline` to make it clear)

Then just compare two folders recursively.